tests/data_quality/test_missing.py

```diff
@@ -323,6 +323,28 @@ def test_results_driver_tenure_values(missing_value_result):
     ]
 
 
-def test_results_driver_tenure_alerts(missing_value_result):
+def test_results_driver_tenure_alerts(missing_value_result):  # noqa: D103
     res = missing_value_result.filter(column_names='driver_tenure').to_df()
     assert list(res[('driver_tenure', 'alert')]) == [False] * 15 + [True] * 5
+
+
+def test_oo_behavior_missing_value_calculator_calculate():  # noqa: D103
```

Comment:
Name could be a bit more descriptive, as this is a weird thing to test.

Maybe something like test_input_dataframes_are_not_altered_by_calculator